ccm_web/package.json

```diff
@@ -41,7 +42,7 @@
     "express": "^4.17.1",
     "express-session": "^1.17.2",
     "got": "^11.8.2",
-    "helmet": "^4.6.0",
+    "helmet": "^5.0.2",
```

Comment:
I chose to adopt the latest major version of `helmet` here, as I started to get confused when looking at the documentation differences between versions. Looks like some good changes to defaults were made in the latest releases: https://github.com/helmetjs/helmet/blob/main/CHANGELOG.md